Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -32,8 +32,8 @@
 #include <QDialog>
 
 #include <boost/foreach.hpp>
-
-
+#include <boost/property_map/property_map.hpp>
```

Comment:
This should be `<CGAL/property_map.h>` to cope with the change of header in boost